src/LambdaFactory.ts

```diff
@@ -74,4 +76,23 @@ export class LambdaFactory implements ILambdaFactory {
       throw e;
     }
   }
+
+  createHttpLambda<ExecuteQueryRequest, ExecuteQueryResults>(lambdaClass: Newable<IHttpEndpoint<ExecuteQueryRequest, ExecuteQueryResults>>, container: IServiceContainer): Handler<APIGatewayProxyEventV2> {
```

Comment:
I'd put this in its own file. It won't function the same as the other lambda factory functions so I think it needs to be stand-alone